plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityEntersBlockScriptEvent.java

```diff
@@ -0,0 +1,87 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityEnterBlockEvent;
+
+public class EntityEntersBlockScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // <entity> goes into <block>
+    //
+    // @Group Entity
+    //
+    // @Location true
+    //
+    // @Cancellable true
+    //
+    // @Triggers when an entity enters, and is stored in a block (EG a bee enters a bee nest)
+    // Does not fire when a silverfish "enters" a stone block. Prefer <@link event entity changes block (into block)> for that.
+    //
+    // @Context
+    // <context.entity> returns the EntityTag.
+    // <context.location> returns the LocationTag of the block entered by the entity.
+    // <context.material> returns the MaterialTag of the block entered by the entity.
+    //
+    // @Player when the entity that entered the block is a player
+    //
+    // @NPC when the entity that entered the block is an NPC.
+    //
+    // -->
+
+    public EntityEntersBlockScriptEvent() {
+        registerCouldMatcher("<entity> goes into <block>");
+    }
+
+    public EntityTag entity;
+    public MaterialTag material;
+    public LocationTag location;
+    public EntityEnterBlockEvent event;
+
+    @Override
+    public boolean matches(ScriptPath path) {
+        String target = path.eventArgLowerAt(0);
+        if (!entity.tryAdvancedMatcher(target)) {
+            return false;
+        }
+        if (!path.tryArgObject(2, material)) {

```

Comment:
Now that you named the event `goes into`, it adds another word and so the number `2` should be incremented to `3` to match the `<block>` parameter and not the word `into`.

Reply:
thank you for catching that - fixed